### ISSUE TITLE
Node JS version needs update to version 18.

### ISSUE THREAD

**Issue by sreekanth684** [awslabs/amazon-kinesis-data-generator#32]
<img width="688" alt="image" src="https://github.com/awslabs/amazon-kinesis-data-generator/assets/14943763/8ab66b36-6b16-43fa-9c72-4c0bedbae4a1">

**Comment by sreekanth684** [comment]
Nope., the template will not run at all with that! Lambda doesn't support that node version any more. Please update that to "nodejs18.x" and that will do it. Thanks.

**Comment by w225** [comment]
Hey, thanks for the reply! Yep I changed the version for nodejs to 18 when
I deployed the KDG and it built up successfully. However, when I try to
send data to the kinesis data stream I created, the KDG will show the pop
up window displaying that records are being sent, while on the kinesis
stream console I cannot see the sent records. Wondering if you have tried
out ingesting into a kinesis data stream and how did it work? Thanks again!
